data/campaigns/World_Conquest/gui/invest_dialog.cfg

```diff
@@ -201,7 +201,7 @@
                                                         vertical_grow = true
                                                         [scroll_label]
                                                             id = "info_label"
-                                                            label = "Text"
+                                                            label = _ "Text"
```

Comment:
This should be an empty string or removed altogether.

Reply:
Replaced it with empty strings.